pilot/pkg/bootstrap/server.go

```diff
@@ -155,12 +156,11 @@ type Server struct {
 	RA             ra.RegistrationAuthority
 
 	// TrustAnchors for workload to workload mTLS
-	workloadTrustBundle *tb.TrustBundle
-	// path to the caBundle that signs the DNS certs. This should be agnostic to provider.
-	caBundlePath string
-	certMu       sync.RWMutex
-	istiodCert   *tls.Certificate
-	server       server.Instance
+	workloadTrustBundle  *tb.TrustBundle
+	certMu               sync.RWMutex
+	istiodCert           *tls.Certificate
+	keyCertBundleWatcher *keycertbundle.Watcher
```

Comment:
I would prefer to call this istiodCertBundleWatcher - this is specifically interested in the cert/key/bundle corresponding to the DNS cert

Reply:
sure